deploy/charts/rook-ceph/values.yaml

```diff
@@ -291,6 +327,8 @@ csi:
     #image: k8s.gcr.io/sig-storage/csi-resizer:v1.4.0
   # Labels to add to the CSI CephFS Deployments and DaemonSets Pods.
   #cephfsPodLabels: "key1=value1,key2=value2"
+  # Labels to add to the CSI CephNFS  DaemonSets Pods.
+  #cephnfsPodLabels: "key1=value1,key2=value2"
```

Comment:
Are these labels for both NFS provisioner and driver, or just driver?

Reply:
> Are these labels for both NFS provisioner and driver, or just driver?

both, similar to what we have for rbd and cephfs